hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/prefetch/TestS3ACachingBlockManager.java

```diff
@@ -244,9 +292,18 @@ private void testPrefetchHelper(boolean forcePrefetchFailure)
       throws IOException, InterruptedException {
     MockS3ARemoteObject s3File = new MockS3ARemoteObject(FILE_SIZE, false);
     S3ARemoteObjectReader reader = new S3ARemoteObjectReader(s3File);
+    BlockManagerParameters blockManagerParamsBuilder =
```

Comment:
looks like the code to create `blockManagerParamsBuilder` is common in these three cases.  Consider moving to a common method that returns the builder.

Reply:
done